native/GTestExample/app/src/androidTest/java/com/example/android/testing/nativesample/AdderTest.kt

```diff
@@ -0,0 +1,9 @@
+package com.example.android.testing.nativesample
+
+import androidx.test.ext.junitgtest.GtestRunner
+import androidx.test.ext.junitgtest.TargetLibrary
+import org.junit.runner.RunWith
+
+@RunWith(GtestRunner::class)
+@TargetLibrary(libraryName = "adder-test")
```

Comment:
What about adding a Java version sample too?

Reply:
That's a good idea, unfortunately there's currently a limitation where only one `GtestRunner`  can exist at a a time so duplicating the test in Java throws an error. It would be great to remove this limitation and I'm hoping to but until then I'm not sure there's a clean way to have both.

I'd also be happy to _replace_ the Kotlin sample with Java is you think that's better. They are nearly identical anyway.